README.md

```diff
@@ -12,7 +12,7 @@ Dependencies
 This layer depends on:
 * poky (http://git.yoctoproject.org/cgit/cgit.cgi/poky)
 * meta-openembedded (http://cgit.openembedded.org/meta-openembedded/)
-* meta-virtualization (http://git.yoctoproject.org/cgit/cgit.cgi/meta-virtualization)
+* meta-virtualization (http://git.yoctoproject.org/cgit/cgit.cgi/meta-virtualization), if the "process-containment" distro feature has been enabled
```

Comment:
this line is getting way too long. I generally prefer lines to be <= 80 chars but we don't have to follow such a short limit (120 maybe?).

Reply:
That would be my preference as well, but this file obviously doesn't follow that rule, given that all paragraphs are on a single line. I'll change this anyway.

Reply:
Created https://github.com/Pelagicore/meta-pelux/pull/37 for this.